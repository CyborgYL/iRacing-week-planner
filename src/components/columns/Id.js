// @flow

import * as React from 'react';

type Props = {
  race: {
    seriesId: number,
  }
};

export default function Id({ race: { seriesId } }: Props) {
  return (
    <td>
      <div>
        {seriesId}
      </div>
    </td>
  );
}
