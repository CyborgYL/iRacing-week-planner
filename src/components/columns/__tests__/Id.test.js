import React from 'react';
import { shallow } from 'enzyme';
import { describe, test } from '@jest/globals';

import Id from '../Id';

describe('components/columns/Id', () => {
  test('renders correctly', () => {
    const component = shallow(<Id race={{ seriesId: 234 }} />);

    expect(component).toMatchSnapshot();
    expect(component.text()).toBe("234");
  });
});
