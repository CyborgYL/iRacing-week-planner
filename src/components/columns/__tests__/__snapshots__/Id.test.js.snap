// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/columns/Id renders correctly 1`] = `
<td>
  <div>
    234
  </div>
</td>
`;
